libcloud/dns/drivers/cloudflare.py

```diff
@@ -436,18 +445,28 @@ def ex_disable_ipv6_support(self, zone):
 
     def _normalize_record_data_for_api(self, type, data):
         """
-        Normalize record data for "special" records such as CAA so it can be
-        used with the CloudFlare API.
+        Normalize record data for "special" records such as CAA and SSHFP
+        so it can be used with the CloudFlare API.
         """
+        cf_data = {}
         if not data:
-            return data
+            return data, cf_data
 
         if type == RecordType.CAA:
             # Replace whitespace with \t character which CloudFlare API
             # expects
             data = data.replace(' ', '\t')
 
-        return data
+        elif type == RecordType.SSHFP:
```

Comment:
How does CloudFlare API returns this data in the list / get records response?

If it returns it inside the ``data`` attribute ant not inside the ``content`` one, ``_normalize_record_data_from_api`` method needs to be updated as well.

Reply:
I quickly verified with their API and it looks like in this case data is indeed returned in the ``content`` field (``"content":"2 1 ABCDEF12345"``) so "de-normalization" is not needed on the response.

I will add a comment to the code to clarify that.